### DISCUSSION TITLE
Internal Error using Help Center

### DISCUSSION THREAD

**Discussion by AlexandreSoftway** [chatwoot/chatwoot#5714] [Self hosted]
I have trouble after update to 2.10 when a try add a article, the system return "Internal Server Error", any friend can help me? Thanks!

the log file! below

[VMLinux.tlp - administrador@20.195.205.151_22 - Bitvise xterm - 20221022-095531376-M0300.txt](https://github.com/chatwoot/chatwoot/files/9844408/VMLinux.tlp.-.administrador%4020.195.205.151_22.-.Bitvise.xterm.-.20221022-095531376-M0300.txt)

And u know how insert image inside of article?

Thanks!!

**Comment by sojan-official** [comment]
It seems like the -slug column is missing in the database. 
Did you run the migrations after updating your chatwoot? 

At the moment image inserts are not supported in articles. it will be available in upcoming releases.
